skopt/gbt.py

```diff
@@ -50,4 +50,4 @@ def fit(self, X, y):
 
     def predict(self, X):
         """Predictions for each quantile."""
-        return np.vstack([rgr.predict(X) for rgr in self.regressors_])
+        return np.asarray([rgr.predict(X) for rgr in self.regressors_])
```

Comment:
Sorry, I had meant that the number of outputs that you get from predict depends on the `quantiles` arguments and not `y` given to fit, which seemed a bit odd.